packages/react-components/src/components/RichTextEditor/RichTextSendBox.test.tsx

```diff
@@ -28,6 +28,88 @@ const icons: {
   chevrondown: <></>
 };
 
+describe('RichTextSendBox should only call onSendMessage when there is content and it is not disabled', () => {
+  beforeAll(() => {
+    registerIcons({
+      icons: icons
+    });
+  });
+  test('onSendMessage should not be called when message is empty', async () => {
+    let called = false;
+    render(
+      <RichTextSendBox
+        onSendMessage={async (): Promise<void> => {
+          called = true;
+          return Promise.resolve();
+        }}
+      />
+    );
+    // Find and click the send button
+    const sendButton = await screen.findByRole('button', {
+      name: 'Send message'
+    });
+    fireEvent.click(sendButton);
+    // Check that onSendMessage was not called
+    expect(called).toBeFalsy();
+  });
+  test('onSendMessage should be called when message is not empty', async () => {
+    let called = false;
```

Comment:
Do we need this test?
It looks very similar to
`HTML string should be correct when send button is clicked` test